scripts/backup/pg_restore.sh

```diff
@@ -49,8 +47,12 @@ if [ "$BACKUP_USER" != "" -a "$(id -un)" != "$BACKUP_USER" ]; then
 fi;
 
 if [ -z $RESTORE_DIR ]; then
-	echo "[ERROR]["$(date +\%Y-\%m-\%d\ %H:%M:%S:%3N)"] Backup not provided, please use -d option." 1>&2
-	exit 1;
+    echo -e "\nRestore directory is empty, retrieving from Google Drive backup"
```

Comment:
@carlosmiranda BTW I didn't find where we install `gdrive`.

Reply:
@vladarefiev there is info in `README.md`, added when #115 was implemented.

Reply:
@carlosmiranda Ah, I thought it will be periodic task. But if it's manual I think it's ok.